test/t8n/t8n.cpp

```diff
@@ -94,11 +100,14 @@ int main(int argc, const char* argv[])
             {
                 j_result["receipts"] = json::json::array();
                 j_result["rejected"] = json::json::array();
-                auto idx = 0;
+                uint64_t idx = 0;
+
                 for (const auto& j_tx : j_txs)
                 {
-                    const auto tx = test::from_json<state::Transaction>(j_tx);
-                    const auto res = state::transition(state, block, tx, rev, vm);
+                    auto tx = test::from_json<state::Transaction>(j_tx);
+                    tx.chain_id = chain_id;
+
+                    auto res = state::transition(state, block, tx, rev, vm);
```

Comment:
It looks like `state::transition()` should compute the logs bloom filter. It feels wrong to patch a receipt here.

Reply:
I will put it there but it will make that every transition call will calculate logs_bloom which is needed only for t8n.